pyproject.toml

```
@@ -0,0 +1,62 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "python-multipart"
+dynamic = ["version"]
+description = "A streaming multipart parser for Python"
+readme = "README.rst"
+license = "Apache-2.0"
+requires-python = ">=3.7"
+authors = [
+    { name = "Andrew Dunham", email = "andrew@du.nham.ca" },
+]
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Environment :: Web Environment',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: Apache Software License',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python :: 3 :: Only',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+]
+dependencies = []
+
+[project.optional-dependencies]
+dev = [
+    "atomicwrites==1.2.1",
+    "attrs==19.2.0",
+    "coverage==6.5.0",
+    "more-itertools==4.3.0",
+    "pbr==4.3.0",
+    "pluggy==1.0.0",
+    "py==1.11.0",
+    "pytest==7.2.0",
+    "pytest-cov==4.0.0",
+    "PyYAML==5.1",
+    "invoke==1.7.3",
+    "pytest-timeout==2.1.0",
+    "hatch",
+]
+
+[project.urls]
+Homepage = "https://github.com/andrew-d/python-multipart"
+Documentation = "https://andrew-d.github.io/python-multipart/"
+Changelog = "https://github.com/andrew-d/python-multipart/tags"
+# Funding = "https://github.com/sponsors/andrew-d"
+Source = "https://github.com/andrew-d/python-multipart"
+
+[tool.hatch.version]
+path = "multipart/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+    "/multipart",
+]
```

- https://hatch.pypa.io/latest/plugins/builder/wheel/#default-file-selection
- https://hatch.pypa.io/latest/plugins/builder/sdist/#default-file-selection

```suggestion
[tool.hatch.build.targets.wheel]
packages = [
    "multipart",
]

[tool.hatch.build.targets.sdist]
include = [
    "/multipart",
]
```

---

I can't find if there's a difference in behavior from the links. Can you expand?

---

I just double checked and the wheel gets generated anyways despite the missing entry on pyproject.toml.
That is, calling `hatch build` with no args generates both a wheel and an sdist.
But hatch docs say [here](https://hatch.pypa.io/latest/build/#building) that:
> Invoking the [build](https://hatch.pypa.io/latest/cli/reference/#hatch-build) command without any arguments will build all **defined** targets, each in an isolated environment:

So i wonder if the docs are outdated or i am misunderstanding something. Appreciate your review btw :)

---

Outdated ty, I'll fix

---

https://github.com/pypa/hatch/commit/5996c9c6ed508ed018248da964481a7efc15bddc